### PR TITLE
use vreg.ymm() instead of ymm(vreg.id())

### PR DESCRIPTION
**Pull request by jspark1105**
Summary: Code refactoring

Differential Revision: D20994679

### CODE REVIEW

File: src/EmbeddingSpMDMNBit.cc [hunk 1]
```diff
@@ -737,34 +713,27 @@ typename ReturnFunctionSignature<indxType, offsetType, ROWWISE_SPARSE>::
...
                   // a YMM as an XMM register.
-                  a->vextracti32x4(
-                      x86::Xmm(temp_vreg.id()), src_vreg, asmjit::Imm(i));
+                  a->vextracti32x4(temp_vreg.xmm(), src_vreg, asmjit::Imm(i));
                 } else {
                   if (i == 1) {
-                    a->pextrq(
-                        scratchReg3_, x86::Xmm(src_vreg.id()), asmjit::Imm(1));
-                    a->movq(x86::Xmm(temp_vreg.id()), scratchReg3_);
+                    a->pextrq(scratchReg3_, src_vreg.xmm(), asmjit::Imm(1));
```
Comment (jspark1105) [inline src/EmbeddingSpMDMNBit.cc:725]: I think there's bug in asmjit. The changes in these lines and lines 731-732 are causing invalid instruction errors. CC @kobalicek

Comment (kobalicek) [reply src/EmbeddingSpMDMNBit.cc:725]: The bug been fixed, thanks!

Comment (dskhudia) [reply src/EmbeddingSpMDMNBit.cc:725]: @jspark1105 We have asmjit fixed to a particular hash in OSS. We will have to update that. For internal, we will need to update asmjit in third-party.

Comment (jspark1105) [reply src/EmbeddingSpMDMNBit.cc:725]: @dskhudia yeah let's defer landing this diff until we update asmjit next time. This diff can definitely wait.

**Pull request closed without merging**